pentasharp/Models/Entities/User.cs

```diff
@@ -39,5 +39,7 @@ public class User
         public ICollection<TaxiBookings> TaxiBookings { get; set; } = new List<TaxiBookings>();
         public ICollection<Notifications> Notifications { get; set; } = new List<Notifications>();
 
+        public bool IsDeleted { get; set; }
```

Comment:
leave one space in between properties and remove data annotations since we agreed on not using them like this anymore